docs/contributing/index.rst

```diff
@@ -5,6 +5,12 @@ Development & community
 django CMS is an open-source project, and relies on its community of users to
 keep getting better.
 
+You can join us online:
+
+* in our IRC channel, #django-cms, on irc.freenode.net
+* our `django CMS users email list <https://groups.google.com/forum/#!forum/django-cms>`_
```

Comment:
You might consider adding a sentence about the purpose of each mailing list, similar to the summaries we have here: https://docs.djangoproject.com/en/dev/internals/mailing-lists/